contracts/interfaces/IAloeBlendImmutables.sol

```diff
@@ -5,20 +5,24 @@ import "./ISilo.sol";
 import "./IVolatilityOracle.sol";
 
 interface IAloeBlendImmutables {
+    /// @notice The nominal time (in seconds) that the primary Uniswap position should stay in one place before
+    /// being recentered
+    function RECENTERING_INTERVAL() external view returns (uint24);
+
     /// @notice The minimum width (in ticks) of the primary Uniswap position
     function MIN_WIDTH() external view returns (uint24);
 
     /// @notice The maximum width (in ticks) of the primary Uniswap position
     function MAX_WIDTH() external view returns (uint24);
 
-    /// @notice The number of standard deviations to +/- from mean when setting primary Uniswap position
-    function B() external view returns (uint8);
-
     /// @notice The maintenance budget buffer multiplier
     /// @dev The vault will attempt to build up a maintenance budget equal to the average cost of incentivization,
     /// multiplied by K.
     function K() external view returns (uint8);
```

Comment:
Is there a concise way of naming this something other than K?

Reply:
Idk about concise. "BUDGET_SIZE" and "BUDGET_MULTIPLIER"?? Fairly long though